packages/dev/docs/pages/blog/creating-a-pointer-friendly-submenu-experience.mdx

```diff
@@ -0,0 +1,108 @@
+
+## Valid Movements
+
+Now let’s imagine two lines: one from the pointer to the top of the submenu, and one from the pointer to the bottom of the submenu. We now essentially have a “range” of where the user might move their pointer on its path to the submenu. Any movement outside of this range should be considered invalid, and the submenu should close if the pointer moves outside of the original menu option.
+
+<SubmenuSafeArea
+  role="img"
+  aria-label="Diagram showing safe area triangle drawn between pointer and submenu" />
+
+## Arctangent
+
+We can measure the angle at which the pointer moved by using the 2-argument arctangent, or [atan2](https://en.wikipedia.org/wiki/Atan2). If we provide our pointer’s delta x and delta y values as arguments, we’ll get the angle (in radians) from the previous pointer position to the current pointer position.
+
+<SubmenuAtan2
+  role="img"
+  aria-label="Diagram showing atan2 value as angle between the X axis and some line on a cartesian plane" />
+
+Now we can use the atan2 function to measure the angles formed by three separate lines:
+
+* **Θ<sub>top</sub>** : Angle formed by the line from the previous pointer position to the **top inside corner** of the submenu
+* **Θ<sub>bottom</sub>** : Angle formed by the line from the previous pointer position to the **bottom inside corner** of the submenu
+* **Θ<sub>pointer</sub>** : Angle formed by the line from the previous pointer position to the **current pointer** position (delta)
+
+<SubmenuAtan2FromPointer
+  role="img"
+  aria-label="Diagram showing triangles formed between pointer and various points on submenu" />
+
+If the pointer’s delta angle is **between** the top and bottom angles, we know the user is moving their pointer in the direction of the submenu.
+
+<p style={{textAlign: 'center', fontSize: 'xx-large'}}>Θ<sub>top</sub> > Θ<sub>pointer</sub> > Θ<sub>bottom</sub></p>
+
+## Time
+
+We also want to consider the pointer’s movement over time when predicting the user’s intent. A user typically increases the **speed** of their pointer when moving towards the submenu, then decreases it as they reach their target. The user might change their mind on the way, however, and slow down to a stop or keep browsing other options in the parent menu. If their pointer speed dips below a certain threshold before arriving at the submenu, we could assume they're no longer intending to go to the submenu.
+We could check the pointer’s speed throughout its journey with our knowledge of what speeds to expect at the different stages along the way. We can even take it a step further by using our knowledge of the [Accot–Zhai steering law](https://en.wikipedia.org/wiki/Steering_law) to predict the time it may take the user to reach the target based on the width of the “tunnel” to the submenu.
```

Comment:
Could you add an additional space between each paragraph. I assume this is what was intended based on the format in this file but in the docs it's just rendering as one giant paragraph and I think it would be nice if it was broken up a little bit.